lib/src/base-styles/_nx-pagination.scss

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-btn-bar--pagination {
+  text-align: center;
+
+  .nx-btn {
+    height: 40px;
+    min-width: 40px;
+    margin: 0 $nx-spacing-xs 0 0;
+    padding: 0 $nx-spacing-md;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    .fa-caret-left, .fa-caret-right {
+      font-size: 20px;
+      vertical-align: -0.2em;
+    }
+  }
+}
+
+.nx-btn--pagination {
+  @include regular();
+  background-color: transparent;
+  border-color: transparent;
+  color: #474747;
+
+  &:hover {
+    background-color: transparent;
+    border-color: #474747;
+  }
+
+  &.selected {
+    background-color: #232c50;
+    border-color: #232c50;
+    color: #fff;
+  }
+
+  &:focus {
+    border: solid 1px #02a6d9;
+    box-shadow: 0 0 3px 1px rgba(2, 166, 217, 0.3);
```

Comment:
I thought we were trying to standardize on using `transparentize` with the hex value for these?

Reply:
Also this is not the default `:focus` state for all buttons so I'm not sure if it would be better to remove it here and implement it to `.nx-btn` as a separate ticket, or if we should implement it in `.nx-btn` here and now. I did create a ticket for the `.nx-btn` `:focus` states here: https://issues.sonatype.org/browse/RSC-384

Reply:
> I thought we were trying to standardize on using transparentize with the hex value for these?

Fixed in 150d6419.  There are now at least three places this exact box-shadow setting is used.  We should probably figure out in what sense it is standard and make a variable for it (maybe even something like `$nx-focus-box-shadow` if it is in fact meant to be used for focus everywhere.

Reply:
> Also this is now the default :focus state for all buttons so I'm not sure if it would be better to remove it here and implement it to .nx-btn as a separate ticket, or if we should implement it in .nx-btn now. I did create a ticket for the .nx-btn :focus states here: https://issues.sonatype.org/browse/RSC-384

Separate ticket I think. And in that ticket the bespoke impl here would get cleaned up.

When did this become the intended focus state for all buttons?  It's feeling like one of those snuck-in-after-the-fact changes to me which indicates there's still a breakdown in our process.

Reply:
> > Also this is now the default :focus state for all buttons so I'm not sure if it would be better to remove it here and implement it to .nx-btn as a separate ticket, or if we should implement it in .nx-btn now. I did create a ticket for the .nx-btn :focus states here: https://issues.sonatype.org/browse/RSC-384
> 
> Separate ticket I think. And in that ticket the bespoke impl here would get cleaned up.
> 
> When did this become the intended focus state for all buttons? It's feeling like one of those snuck-in-after-the-fact changes to me which indicates there's still a breakdown in our process.

I think that @dturner32 was asked to make it the default a little while ago when the `:focus` states settled down into a consistent state.

Reply:
> > > Also this is now the default :focus state for all buttons so I'm not sure if it would be better to remove it here and implement it to .nx-btn as a separate ticket, or if we should implement it in .nx-btn now. I did create a ticket for the .nx-btn :focus states here: https://issues.sonatype.org/browse/RSC-384
> > 
> > 
> > Separate ticket I think. And in that ticket the bespoke impl here would get cleaned up.
> > When did this become the intended focus state for all buttons? It's feeling like one of those snuck-in-after-the-fact changes to me which indicates there's still a breakdown in our process.
> 
> I think that @dturner32 was asked to make it the default a little while ago when the `:focus` states settled down into a consistent state.

yes, I added the shadow for the focus state a while back. Glen and I had a conversation around this because the focus border on its own wasn't prominent enough in some cases.

Reply:
> Separate ticket I think. And in that ticket the bespoke impl here would get cleaned up.

That other ticket has been merged to master and this PR now relies on the default button focus styles.  Note that an additional issue has been found with button focus styles which I have put in as another ticket: https://issues.sonatype.org/browse/RSC-391

Reply:
> Note that an additional issue has been found with button focus styles which I have put in as another ticket: https://issues.sonatype.org/browse/RSC-391

RSC-291 is now Done and merged into this branch as well.